pkgs/top-level/all-packages.nix

```diff
@@ -27215,6 +27215,10 @@ in
 
   vokoscreen = libsForQt5.callPackage ../applications/video/vokoscreen { };
 
+  vokoscreen-ng = libsForQt5.callPackage ../applications/video/vokoscreen-ng {
```

Comment:
Should the attribute and directory be “vokoscreenNG” rather than “vokoscreen-ng”? (same as the “pname”, the commit message, the name used upstream etc.)

Reply:
All of them should be lower case, and the commit message and PR title should refer to the attribute name which is lower case.

Reply:
What about the dash?

Reply:
Pname is matching the repo name on github. Attribute name is in snake case to follow the convention. But if the convention is the have the pname matching the attribute name, I can make everything snake case : commit message, attribute name and pname.

Reply:
Yes please. Also, squash the commits.

Reply:
Fixed.

Reply:
Awesome :+1:.